tests/unit/lms/views/decorators/lis_result_sourcedid_test.py

```diff
@@ -46,8 +46,9 @@ def test_it_raises_if_validation_raises_other_than_ValidationError(
 
         with pytest.raises(TypeError, match="foo"):
             upsert_lis_result_sourcedid(context, pyramid_request)
-            lis_result_sourcedid_svc.upsert.assert_not_called()
-            wrapped.assert_called_once_with(context, pyramid_request)
+
+        lis_result_sourcedid_svc.upsert.assert_not_called()
+        wrapped.assert_not_called()
```

Comment:
This was really crazy, and not getting called at all. As a result the assertions didn't actually pass